roles/openshift_common/meta/main.yml

```diff
@@ -13,4 +13,4 @@ galaxy_info:
   - cloud
 dependencies:
 - role: openshift_facts
-- role: openshift_repos
+- role: openshift_docker
```

Comment:
This was done because I now needed the base package for versioning installation in openshift_docker (now called openshift_version). I could make this a dep on openshift_version now instead which should dodge useless docker installation, or just duplicate the two tasks involved.